variants/STM32G0xx/G030J6M/generic_clock.c

```diff
@@ -20,8 +20,43 @@
   */
 WEAK void SystemClock_Config(void)
 {
-  /* SystemClock_Config can be generated by STM32CubeMX */
-#warning "SystemClock_Config() is empty. Default clock at reset is used."
+  RCC_OscInitTypeDef RCC_OscInitStruct = {0};
+  RCC_ClkInitTypeDef RCC_ClkInitStruct = {0};
```

Comment:
Please remove `0` to avoid compilation warning.
The same for all clock configuration.

Reply:
This needs to be done on both

Reply:
No worries, i will try to do that now. sorry if multiple Pull requests have been made, I though there was another button i needed to select to submit (and went looking multiple times for it)

Reply:
Not sure why the 0's and Readme issues are still there, i checked those fixes in, in the last hour

Reply:
apologies for the issues i'm causing, i'm still trying to get used to GitHub when working with Pull Requests. Hopefully i will get the hang of it soon